src/styles/variables.ts

```diff
@@ -167,7 +167,7 @@ export default {
     eReceiptWordmarkWidth: 86,
     eReceiptBGHeight: 540,
     eReceiptBGHWidth: 335,
-    reportPreviewMaxWidth: 302,
+    reportPreviewMaxWidth: 335,
```

Comment:
So should this be same for report preview and money request action now and be `335` or the previous `302`?

Reply:
Yes, same constant. Let's use `335` for now.

Reply:
I have a PR updating this flow for eReceipts that needs a design pass, so if this number needs to be changed we can handle it there. Let's just make sure we test both report previews and money request previews with this PR

Reply:
Perf, thanks!